### ISSUE TITLE
Tracking non persistent events

### ISSUE THREAD

**Issue by aerosol** [ZennerIoT/ex_audit#15]
Hello,

What would be the suggested way of (ab)using this library so that events that don't persist changes in the database are tracked along with those that do?

For example, track every "show" action for certain entities.

If this is a feature that could be scoped within this library I'd appreciate any pointers so I can submit a patch for review. 

The simplest approach I can think of would be to record an empty `Version.patch` with a new (custom?) `ExAudit.Type.Action`. Would that make any sense?

**Comment by narrowtux** [comment]
Yes it would definitely fit into the concept when you have an empty patch and a custom action type. 

The custom action types could also be used for actual patches that convey a different meaning than just "updated".
